src/components/CreateCourseRunPage/index.jsx

```diff
@@ -83,6 +86,18 @@ class CreateCourseRunPage extends React.Component {
       ({ label: choice.display_name, value: choice.value }));
   }
 
+  compareDates(a, b) {
```

Comment:
Don't love the a, b. Maybe even `dateOne` and `dateTwo` or something. Also, looking at what this is doing, it is misleading to say it is comparing dates because it isn't just comparing two dates. It is specifically comparing two start dates on an object that includes a key called start and that field is a date.

Reply:
I think this should also have a docstring to explain what this is going to return. It's unclear from the function name and the code inside the function. This might also be a good opportunity to use an arrow function definition. I think it would look something like 
```
const compareDates = (a, b) => (moment(b.start).diff(moment(a.start)))
```

Reply:
Do you think it'd be more appropriate to do something like this?

```
parseCourseRunLabels(courseRuns) {
    // Sort course runs by descending start dates
    const sortedCourseRuns =
      courseRuns.sort((run1, run2) => moment(run2.start).diff(moment(run1.start)));
```

Or maybe this:
```
// Helper method used to compare course runs and sort by descending start dates
compareRunDates = (run1, run2) => (moment(run2.start).diff(moment(run1.start)));

parseCourseRunLabels(courseRuns) {
    const sortedCourseRuns = courseRuns.sort(this.compareRunDates);
```

I can't tell what the tradeoffs are between these two.

Reply:
And do you think I need to describe what that `moment().diff();` is returning? I'm not sure if the Array.sort() parameters are common knowledge..

Reply:
I think I prefer the top one, unless you think this compareRunDates function could be used in other parts of PubFE, in which case I'd argue for putting it in a util file. If you do that, I'd just write comments in both places.

Also, if you keep compareRunDates, please rename to compareRunStartDates since it doesn't compare any other dates. Or abstract it so it can compare any two dates (and if you do this, maybe also add in a parameter to specify ascending vs descending order). I don't have a preference either way.

Re needing to describe: Eh, I think your comment does enough. If someone needs to know more, they can look up the docs. You describe the functionality of your code by saying that you're sorting course runs by descending start dates. Would be overkill to copy moment.js docs or Array.prototype docs.